src/components/NavBar/NavBar.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { BrowserRouter as Router, Link } from 'react-router-dom';
+import Icons from '../Icons/FAIcon';
+import './NavBar.css';
+import logo from './Logo/Logo.png';
+
+const NavBar = () => (
+    <Router>
+        <div>
+            <nav className="nav-bar">
+                <div className="mobile-style">
+                    {/* Replace with burger component when complete */}
+                    <Link to="#">
+                        <Icons name={['fas', 'bars']} size="2x" />
+                    </Link>
+                </div>
+                <div className="mobile-style">
+                    {/* Replace with logo component */}
+                    <img className="mobile-logo" src={logo} alt="Modern trousseau logo" />
```

Comment:
Can you please import the logo from Component Library here instead of the one you are using.

Reply:
Imported logo from Component Library